lib/restbase.js

```diff
@@ -79,6 +80,24 @@ RESTBase.prototype.makeChild = function(req) {
     return child;
 };
 
+// Create a json web token
+// @param {string} token
+// @return {string} JWT signed token
+rbUtil.encodeToken = function(token) {
+    console.log(self._priv.options);
+    var newToken = jwt.sign(result.meta.pageState,
+                        self._priv.options.saltKey &&
+                        self._priv.options.saltKey.toString());
+    return newToken;
+}
+
+// Decode signed token and decode the orignal token
+// @param {string} JWT token
+// @return {string} original token
+rbUtil.decodeToken = function(token) {
+    console.log(self._priv.options);
+    return jwt.verify(token, self._priv.options.saltKey)
```

Comment:
This will throw an exception on failure, which we should report with HTTP status 400 (by throwing an appropriate `HTTPError`). It might make sense to catch & re-wrap the error right here, so that we don't have to do that at each call site.